core/templates/pages/moderator-page/moderator-page.component.ts

```diff
@@ -16,124 +16,145 @@
  * @fileoverview Component for the Oppia moderator page.
  */
 
+import { ChangeDetectorRef, Component } from '@angular/core';
+import { downgradeComponent } from '@angular/upgrade/static';
+import { AppConstants } from 'app.constants';
 import { ThreadMessage } from 'domain/feedback_message/ThreadMessage.model';
+import _ from 'lodash';
+import { AlertsService } from 'services/alerts.service';
+import { DateTimeFormatService } from 'services/date-time-format.service';
+import { LoaderService } from 'services/loader.service';
+import { Schema } from 'services/schema-default-value.service';
+import { ActivityIdTypeDict, CommitMessage, ExplorationDict, ModeratorPageBackendApiService } from './services/moderator-page-backend-api.service';
 
 require('base-components/base-content.directive.ts');
-require(
-  'components/forms/schema-based-editors/' +
-  'schema-based-editor.directive.ts');
-
-require('services/alerts.service.ts');
-require('services/date-time-format.service.ts');
-
-angular.module('oppia').component('moderatorPage', {
-  template: require('./moderator-page.component.html'),
-  controller: [
-    '$http', 'AlertsService', 'DateTimeFormatService', 'LoaderService',
-    'ENTITY_TYPE',
-    function(
-        $http, AlertsService, DateTimeFormatService, LoaderService,
-        ENTITY_TYPE) {
-      var ctrl = this;
-      ctrl.getDatetimeAsString = function(millisSinceEpoch) {
-        return DateTimeFormatService.getLocaleAbbreviatedDatetimeString(
-          millisSinceEpoch);
-      };
-
-      ctrl.isMessageFromExploration = (
-        (message) => message.entityType === ENTITY_TYPE.EXPLORATION);
-
-      ctrl.getExplorationCreateUrl = function(explorationId) {
-        return '/create/' + explorationId;
-      };
-
-      ctrl.getActivityCreateUrl = function(reference) {
-        var path = (
-          reference.type === ENTITY_TYPE.EXPLORATION ?
+
+@Component({
+  selector: 'oppia-moderator-page',
+  templateUrl: './moderator-page.component.html'
+})
+export class ModeratorPageComponent {
+  activeId: number = 1;
```

Comment:
WHy is this initialized to one and where is it used?

Reply:
removed.